scripts/semconv/templates/semantic_attributes.j2

```diff
@@ -42,11 +42,12 @@ Note: {{ common.to_docstring(attribute.note | indent)}}.
 """
 {%- endmacro -%}
 
-{%- if root_namespace not in excluded_namespaces -%}
+{%- if root_namespace not in excluded_namespaces.split(' ') -%}
```

Comment:
this fixes a bug that resulted in missing `net` and `os` attributes,

not strictly related to this change